src/reports/logic.py

```diff
@@ -12,6 +13,68 @@ def take(n, iterable):
     "Return first n items of the iterable as a list"
     return list(islice(iterable, n))
 
+#
+#
+#
+
+def status_report():
+    "useful insights into the current state of this lax instance"
+
+    #al = models.Article.objects.all()
+    avl = models.ArticleVersion.objects \
+        .select_related('article') \
+        .defer('article_json_v1', 'article_json_v1_snippet') \
+        .order_by('article__manuscript_id', 'version') \
+        .all()
+
+    def get_loc(av):
+        try:
+            return av.article.articlefragment_set.get(type=models.XML2JSON).fragment['article']['-meta']['location']
+        except models.ArticleFragment.DoesNotExist:
+            return 'no-article-fragment'
+        except KeyError:
+            return 'no-location-stored'
+
+    def avi(av):
```

Comment:
some docstrings to understand what `avi` and `avil` is the acronym of would be helpful